### PR TITLE
feat: hooks config

### PR DESCRIPTION
**Pull request by sand4rt**
related to: https://github.com/microsoft/playwright/pull/18616

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -30,6 +32,27 @@ _addRunnerPlugin(() => {
   });
 });
 
-const test = baseTest.extend(fixtures);
+type JsonPrimitive = string | number | boolean | null;
+type JsonValue = JsonPrimitive | JsonObject | JsonArray;
+type JsonArray = JsonValue[];
+type JsonObject = { [Key in string]?: JsonValue };
```
Comment (JoviDeCroock) [inline src/index.ts]: Can't we re-use these types as they are also used in `src/hooks.mts`

**Review by JoviDeCroock — APPROVED** [review]
Left a minor nit, great work!

File: src/index.ts (hunk 1 above)
Comment (sand4rt) [reply src/index.ts]: Is this what you were thinking about?: [chore: reuse hooks config type](https://github.com/preactjs/playwright-ct/pull/6/commits/db68f970047c1cde1874964e1a8294bf2080bc5c)

**Comment by sand4rt** [comment]
Then it's easier to integrate it into Playwright's repository. Perhaps we should create an issue first to ask if they are ready or if we have to wait a little longer? All features match with Playwright's features, so IMO it's about time. What do you think?

**Pull request merged**